boards/arm/stm32l1_disco/stm32l1_disco.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: stm32l1_disco
+name: STM32L1DISCOVERY
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 16
+flash: 128
+supported:
+  - gpio
```

Comment:
- i2c
  - spi

Reply:
sure, I've not tested (and can't test) i2c, but I got some spi traffic out of it.

Reply:
Ok... This will at least allow sanitycheck compilation. Better than nothing